src/modules/fetchLikes.js

```diff
@@ -0,0 +1,13 @@
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
- [ ] KIndly remove this & solve the linter manually, if the error present.

Reply:
well resolved! ✔️